addons/mail/static/tests/discuss_app/discuss.test.js

```diff
@@ -404,15 +406,18 @@ test("Can reply to starred message", async () => {
     await contains(".o_notification", { text: 'Message posted on "RandomName"' });
     await click(".o-mail-DiscussSidebarChannel", { text: "RandomName" });
     await contains(".o-mail-Message-content", { text: "abc" });
+    await contains(".o-mail-Message.o-selfAuthored a.o_mail_redirect", { count: 0 });
```

Comment:
Same remark for assertion of `count: 0` without matching positive one.
I don't think this assertion is that useful.